recipes/litehtml/all/conandata.yml

```
@@ -0,0 +1,4 @@
+sources:
+  "cci.28102021":
```

We use `ccci.YYYYMMDD` format to make ordering cci versions easier.
```suggestion
  "cci.20211028":
```